src/buffers.h

```diff
@@ -0,0 +1,18 @@
+#ifndef SRC_BUFFERS_H_
+#define SRC_BUFFERS_H_
+
+// Translated into UTF-8, this means that a worst-case UTF-16
+// buffer would be double (4 bytes).
+#define MAX_UTF8_BUFFER 256
+
+// We frequently have terminator characters with the UTF-8 buffers.
+#define MAX_UTF8_BUFFER_WITH_TERM (MAX_UTF8_BUFFER + 1)
```

Comment:
I'm not seeing this constant used anywhere else.

Reply:
Removed.